src/Utils.ts

```diff
@@ -640,3 +589,27 @@ function strVar(str: string, vars: { [name: string]: any }): string {
   }
   return str
 }
+
+/**
+ * Searches for a given string in a given array of values
+ * @param needle What string to search for
+ * @param haystack Array where to search
+ * @returns Indices that match the search
+ */
+function matchString(needle: string, haystack: string[]): number[] {
+  if (haystack.length === 0) { return [] }
+  haystack = ufuzzy.latinize(haystack)
+  const arr = []
+  const idxs = uf.filter(haystack, needle)
+  if (idxs != null && idxs.length > 0) {
+    const infoThresh = 1e4
+    if (idxs.length <= infoThresh) {
+      const info = uf.info(idxs, haystack, needle)
+      const order = uf.sort(info, haystack, needle)
```

Comment:
Maybe change variable names here as idxs isn't the best one.

Reply:
probably will rename them but tbh it matters not